docs/development/setup-recommended.md

```diff
@@ -317,7 +317,22 @@ cd zulip
 vagrant up --provider=docker
 ```
 
-The first time you run this command it will take some time because vagrant
+If you are using macOS, you may see something like this:
+```bash
```

Comment:
This isn't a shell script, just standard output, so no need for a language specifier here. (just removing the word `bash` is fine: I can't use GitHub's suggestions feature here due to a syntax quirk 😃)